libraries/botbuilder-dialogs-adaptive/src/adaptiveComponentRegistration.ts

```diff
@@ -189,7 +189,7 @@ type Type<T> = {
  * `ComponentRegistration` implementation for adaptive components.
  */
 export class AdaptiveComponentRegistration extends ComponentRegistration implements ComponentDeclarativeTypes {
-    private _declarativeTypes: DeclarativeType<unknown, unknown>[] = [];
+    protected _declarativeTypes: DeclarativeType<unknown, unknown>[] = [];
```

Comment:
Is this mirroring a change in .NET? The property name should drop the name-mangling since the access modifier is protected.

Reply:
Accidental change. Reverting.